uni/ulsp/server.icn

```diff
@@ -251,9 +271,54 @@ class Server(
       jrpc_handler.send_msg(msg)
    end
 
+########################################################
+#                textDocument/documentSymbol           #
+########################################################
+
+   method symbolHelp(msg)
+      local filehandler_object, uri, result
+      
+      if uri := msg.get_param_uri() then {
```

Comment:
This will always succeed. So I think you still need a no null check ` \(uri := msg.get_param_uri()) `